src/code42cli/cmds/search/extraction.py

```diff
@@ -96,3 +107,9 @@ def create_time_range_filter(filter_cls, begin_date=None, end_date=None):
 
     elif end_date and not begin_date:
         return filter_cls.on_or_before(end_date)
+
+
+def _process_events(output_format, include_all, events, output_header):
+    if include_all:
```

Comment:
Now... I don't think we need to print the header again fir the next page.

The experience should be that we are paging through the records in the terminal and shouldn't be able to distinguish when the next set of events (the HTTP page) comes through... The header should happen only once I think.

That would required another condition checked here, like `not is_first_page`... I am pretty sure the HTTP response will tell us what page it is. If not, there are other ways.

Reply:
Ah, nevermind about it being in the response. I forgot that this is not actually an API paging mechanism

Reply:
Hmm... You can the extractor's `_self._previous_event_count` though

Reply:
None of what I said here is relevant anymore.... https://github.com/code42/code42cli/pull/133#issuecomment-674169518